addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2073,6 +2073,24 @@ const Wysiwyg = Widget.extend({
 
         return finalOptions;
     },
+    _getBannerCommand: function (title, alertClass, iconClass, description, priority) {
+        return {
+            category: _t('Banner'),
+            name: title,
+            priority: priority,
+            description: description,
+            fontawesome: iconClass,
+            isDisabled: () => !this.odooEditor.isSelectionInBlockRoot(),
+            callback: () => {
+                const bannerHTML = $(QWeb.render('web_editor.banner', {
```

Comment:
I find the name confusing with `innerHTML` and `outerHTML`, here in `bannerHTML` you have an `Element`, not a `string` representing `HTML`, but maybe it's just me, feel free to ignore!

Reply:
sure `bannerElement` seems fine :)